expansionTest.file.spec.js

```diff
@@ -0,0 +1,20 @@
+it('visit GU website', () => {
+    cy.visit('/')
+    cy.get('cerberus-navbar-main-menu-item gu-simple-text').contains('expansion').click()
+})
+
+it('expansion'), () => {
```

Comment:
Lines 6 - 18 don't seem to have assertions, which check for the correct conditions.

Reply:
Lines 6 - 18 are also testing 'how to use env variables' which I'm not sure is relevant.